x-pack/legacy/plugins/lens/public/indexpattern_plugin/operations/definitions/date_histogram.test.tsx

```diff
@@ -118,6 +132,27 @@ describe('date_histogram', () => {
     };
   });
 
+  function stateWithInterval(interval: string) {
+    return ({
+      ...state,
+      layers: {
+        ...state.layers,
+        first: {
+          ...state.layers.first,
+          columns: {
+            ...state.layers.first.columns,
+            col1: {
+              ...state.layers.first.columns.col1,
+              params: {
+                interval,
+              },
+            },
+          },
+        },
+      },
+    } as unknown) as IndexPatternPrivateState;
```

Comment:
...wow

Reply:
lol. I'm open to alternatives. This is an example of where TypeScript isn't super helpful.